Automattic/jetpack
# Move Salesforce lead form to production

#### Changes proposed in this Pull Request:
Make Salesforce Lead form available in production

#### Other information:


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Edit a post and add a form. You should see the Salesforce Lead form available on the variation picker without the need to enable beta blocks
